.circleci/config.yml

```diff
@@ -404,6 +416,106 @@ references:
     - Demisto SDK Nightly - Run against Cortex XSOAR:
         context: sdk_nightly_env
 
+  bucket_upload_jobs: &bucket_upload_jobs
+    - Setup Environment
+    - Create Instances:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Run Unit Testing And Lint:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Run Validations:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Install Packs in Demisto 6_0:
+        requires:
+          - Create Instances
+    - Install Packs in Demisto Marketplace - Master:
+        requires:
+          - Create Instances
+    - Upload Packs:
+        context: bucket_upload_env
+        requires:
+          - Run Unit Testing And Lint
+          - Run Validations
+          - Install Packs in Demisto 6_0
+          - Install Packs in Demisto Marketplace - Master
+
+
+  upload_packs_to_marketplace_storage: &upload_packs_to_marketplace_storage
```

Comment:
Things that are written only once should not be in references, write it in the job steps directly.